frontend/src/lib/k8s/index.ts

```diff
@@ -86,9 +86,19 @@ classList.forEach(cls => {
 export const ResourceClasses = resourceClassesDict;
 
 // Hook for getting or fetching the clusters configuration.
-export function useClustersConf(): ConfigState['clusters'] {
-  const clusters = _.cloneDeep(useTypedSelector(state => state.config.clusters));
-  return clusters;
+export function useClustersConf(): ConfigState['allClusters'] {
```

Comment:
This seems to be only used once, so it should live near where it's used.

Reply:
I am sorry I don't understand what you mean my used only once? I workflow is we are combining normal clusters and stateless clusters. Scenario can be user don't have it enabled then it will only return allClusters as normal clusters from all sources. But if it is enabled then we combine the two.

Also `useClustersConf` is used throught the frontend. Please correct me if I am missing something?

cc @illume